projects/js-packages/shared-extension-utils/.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+const loadIgnorePatterns = require( '../../../tools/js-tools/load-eslint-ignore.js' );
+
+module.exports = {
+	// This project uses react, so load the shared react config.
+	root: true,
+	extends: [ '../../../.eslintrc.react.js' ],
```

Comment:
Would this break when importing this package anywhere outside the monorepo? (same for the `require` on line 1).

Reply:
Importing the package anywhere outside the monorepo will mean publishing it to a central registry like npm first and at that time it should have its own build step which imo should take care of this.
But yeah if we move the package codebase somewhere else then it will certainly break.

Reply:
Right, that makes sense. Seems fine to leave it like this for now then 👍🏼